package.json

```diff
@@ -114,6 +114,7 @@
     "sinon-chai": "^3.5.0",
     "strict-event-emitter-types": "^2.0.0",
     "terser": "^3.17.0",
```

Comment:
`terser-webpack-plugin` looks to use `terser` 4.x - should this be updated/removed?

Reply:
good catch